spring-cloud-stream-binder-kafka/src/test/java/org/springframework/cloud/stream/binder/kafka/KafkaBinderJaasInitializerListenerTest.java

```diff
@@ -57,5 +60,10 @@ public void testConfigurationParsedCorrectlyWithKafkaClient() throws Exception {
 	@SpringBootApplication
 	public static class SimpleApplication {
 
+		@Bean
+		BindingServiceProperties bindingServiceProperties() {
```

Comment:
I removed those mock bindingServiceProperties beans in https://github.com/spring-cloud/spring-cloud-stream-binder-kafka/pull/143/commits/4c6db2be73249550363d4f852667ceb298d76dff, as we don't rely on this bean anymore on the behalf of `TopicInformation#consumerGroup`.